src/lib/y2users/linux/writer.rb

```diff
@@ -124,15 +130,38 @@ def write
       CHPASSWD = "/usr/sbin/chpasswd".freeze
       private_constant :CHPASSWD
 
+      # Whether given user does not exist yet
+      #
+      # FIXME: choose a better criteria to identify a new user. An internal Y2User::User id?
+      #
+      # @param user [Y2User::User]
+      # @return [Boolean] true if there is a user with same name in @initial_config; false otherwise
+      def new_user?(user)
+        @initial_users_names ||= initial_config.users.map(&:name)
+
+        !@initial_users_names.include?(user.name)
```

Comment:
Self note: this check should be reviewed and, most probably, reviewed. Looks like the username can be changed too.

Reply:
See https://github.com/yast/yast-users/issues/257